lib/omniauth/strategies/azure_activedirectory_v2.rb

```diff
@@ -14,41 +16,45 @@ class AzureActivedirectoryV2 < OmniAuth::Strategies::OAuth2
       args [:tenant_provider]
 
       def client
-        if options.tenant_provider
-          provider = options.tenant_provider.new(self)
-        else
-          provider = options  # if pass has to config, get mapped right on to options
-        end
+        provider = if options.tenant_provider
+                     options.tenant_provider.new(self)
+                   else
+                     options # if pass has to config, get mapped right on to options
+                   end
 
         options.client_id = provider.client_id
         options.client_secret = provider.client_secret
         options.tenant_id =
-            provider.respond_to?(:tenant_id) ? provider.tenant_id : 'common'
+          provider.respond_to?(:tenant_id) ? provider.tenant_id : 'common'
         options.base_azure_url =
-            provider.respond_to?(:base_azure_url) ? provider.base_azure_url : BASE_AZURE_URL
+          provider.respond_to?(:base_azure_url) ? provider.base_azure_url : BASE_AZURE_URL
 
         options.authorize_params = provider.authorize_params if provider.respond_to?(:authorize_params)
-        options.authorize_params.domain_hint = provider.domain_hint if provider.respond_to?(:domain_hint) && provider.domain_hint
-        options.authorize_params.prompt = request.params['prompt'] if defined? request && request.params['prompt']
-        options.authorize_params.scope = (provider.scope if provider.respond_to?(:scope) && provider.scope) || DEFAULT_SCOPE
+        if provider.respond_to?(:domain_hint) && provider.domain_hint
+          options.authorize_params.domain_hint = provider.domain_hint
+        end
+        options.authorize_params.prompt = request.params['prompt'] if defined?(request) && request.params['prompt']
+        options.authorize_params.scope = (if provider.respond_to?(:scope) && provider.scope
+                                            provider.scope
+                                          end) || DEFAULT_SCOPE
 
         options.client_options.authorize_url = "#{options.base_azure_url}/#{options.tenant_id}/oauth2/v2.0/authorize"
         options.client_options.token_url = "#{options.base_azure_url}/#{options.tenant_id}/oauth2/v2.0/token"
 
         super
       end
 
-      uid {
+      uid do
         raw_info['oid']
-      }
+      end
```

Comment:
Both the Google OAuth 2 and Xero OAuth 2 OmniAuth plugins use this form:

```
uid { ...str... }
```

...so if this was reformatted at all, I'd prefer to see it go single-line and keep the curly braces at that appears to be the more idiomatic form for 3rd party OmniAuth strategy gem code.